spec/system/user_preferences_interface_spec.rb

```diff
@@ -9,32 +9,26 @@
 
   describe "Bookmarks" do
     it "changes the bookmark after notification preference" do
-      skip(<<~TEXT) if ENV["CI"]
-      This is currently failing on CI with the following:
+      user_preferences_page.visit(user)
+      click_link(I18n.t("js.user.preferences_nav.interface"))
 
-      ```
-      Failure/Error: expect(page).to have_content(I18n.t("js.saved"))
-      expected `#<Capybara::Session>.has_content?("Saved!")` to be truthy, got false
-      ```
-      TEXT
-
-      user_preferences_page.visit(user).click_interface_tab
+      dropdown = PageObjects::Components::SelectKit.new("#bookmark-after-notification-mode")
 
       # preselects the default user_option.bookmark_auto_delete_preference value of 3 (clear_reminder)
-      expect(user_preferences_interface_page).to have_bookmark_after_notification_mode(
-        Bookmark.auto_delete_preferences[:clear_reminder],
-      )
-
-      user_preferences_interface_page.select_bookmark_after_notification_mode(
-        Bookmark.auto_delete_preferences[:when_reminder_sent],
-      ).save_changes
-
-      expect(
-        UserOption.exists?(
-          user_id: user.id,
-          bookmark_auto_delete_preference: Bookmark.auto_delete_preferences[:when_reminder_sent],
-        ),
-      ).to eq(true)
+      expect(dropdown).to have_selected_value(Bookmark.auto_delete_preferences[:clear_reminder])
+
+      dropdown.select_row_by_value(Bookmark.auto_delete_preferences[:when_reminder_sent])
+      click_button(I18n.t("js.save"))
+
+      # the preference page reloads after saving, so we need to poll the db
```

Comment:
This is a regression, the page no longer reloads after saving so there is no need to reload the page. I don't really thing the `try_until_success` pattern is good to follow because we're unnecessarily hammering the database and a timeout of 20 seconds here is unnecessary.

Reply:
Yes I also think we should never have to check for objects in DB from system specs